laws/src/main/scala/org/http4s/laws/EntityEncoderLaws.scala

```diff
@@ -23,7 +23,7 @@ import cats.laws._
 import org.http4s.headers.{`Content-Length`, `Transfer-Encoding`}
 
 trait EntityEncoderLaws[F[_], A] {
-  implicit def F: Sync[F]
+  implicit def F: Effect[F]
```

Comment:
I need `Effect` as we have to eventually evaluate on `IO` to use `munit-cats-effect` assertions